projects/m2k/common/m2k_bd.tcl

```diff
@@ -278,7 +278,6 @@ ad_connect trigger_t adc_trigger/trigger_t
 
 ad_connect axi_ad9963/dac_sync_in axi_ad9963/dac_sync_out
 ad_connect axi_ad9963/adc_dovf    ad9963_adc_dmac/fifo_wr_overflow
-ad_connect axi_ad9963/dac_dunf    ad9963_dac_dmac_a/fifo_rd_underflow
```

Comment:
Shouldn't we re-create the underflow detection from the interpolate core ?  What happens if the DMACs de-asserts 'valid' after the trigger,

Can we add in the axi_dac_interpolate_filter.v something like 
underflow =  transfer & ~dma_valid
or something similar ?

Reply:
I will add it back. Thanks :)